src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +46,10 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(s -> s.equals(element))
+                .findAny()
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
look closely to orElseThrow() without parameters)

Reply:
Fixed!